Source.md

```diff
@@ -1,12 +1,25 @@
 ---
 title: Flutter Interview Questions
 ---
+* What is the difference between Stateless and Stateful Widget in Flutter? </br>
+   If a widget can change—when a user interacts with it, for example—it’s stateful.
```

Comment:
Hello! I believe the purpose of this repository is just to post questions and not answers about flutter interviews.